.rubocop.yml

```diff
@@ -6,7 +6,7 @@ require:
   - rubocop-rspec
 
 AllCops:
-  TargetRubyVersion: 2.4
+  TargetRubyVersion: 2.5
```

Comment:
Needed to bump this to allow block-level `rescue` [here](https://github.com/sul-dlss/dlme/pull/778/files#diff-f9ea5052e1fd801bed3e93c8f3fe10d6R27)